Sources/BrowserServicesKit/ContentBlocking/ContentBlockerRulesManager.swift

```diff
@@ -177,27 +183,29 @@ public class ContentBlockerRulesManager {
 
     public init(rulesSource: ContentBlockerRulesListsSource,
                 exceptionsSource: ContentBlockerRulesExceptionsSource,
+                cache: ContentBlockerRulesCaching? = nil,
```

Comment:
It would be good to include tests for this - in case there is cache, or nothing is provided, to ensure we are clearing things properly.

Reply:
Added tests